index.html

```diff
@@ -202,6 +202,12 @@ <h1>The Registration Process</h1>
 description of the addition.
       </li>
       <li>
+The name of the extension MUST be indicative of its function, or - in the case
+of DID Methods - of the nature of its underlying verifiable data registry or
+the company or community that proposes it. E.g. avoid properties such as
+"myProperty" or DID Methods such as "identity" or "registry".
```

Comment:
I think there's something good here, but I can't quite figure out what it is. Can you point to specific places in the current registry that run afoul of this rule?

Reply:
did:schema ?

I think its reasonable advice.

Reply:
Clearly this is targeted at the discussion on this PR as well: https://github.com/w3c/did-spec-registries/pull/363

I don't feel strongly about name squatting, or trying to get 'really good did method names" like "3" or "id" or "schema"...

However, if they are abandoned it should be possible to reclaim them, just as it is for domains, github and twitter handles.... otherwise somebody can grab all the good ones, and the community looses access to them forever.

Reply:
I opened https://github.com/w3c/did-spec-registries/issues/374 so we keep that thread separate from this one.